source/user-manual/ruleset/custom.rst

```diff
@@ -106,34 +103,30 @@ Here's an example on how to change the level value of the SSH rule ``5710`` from
 
    .. code-block:: xml
 
-      <group name="syslog,sshd,">
-        <rule id="5710" level="5">
-          <if_sid>5700</if_sid>
-          <match>illegal user|invalid user</match>
-          <description>sshd: Attempt to login using a non-existent user</description>
-          <mitre>
-            <id>T1110</id>
-          </mitre>
-          <group>invalid_login,authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,pci_dss_10.6.1,gpg13_7.1,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,nist_800_53_AU.6,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
-        </rule>
-      </group>
+      <rule id="5710" level="5">
+        <if_sid>5700</if_sid>
+        <match>illegal user|invalid user</match>
+        <description>sshd: Attempt to login using a non-existent user</description>
+        <mitre>
+          <id>T1110</id>
+        </mitre>
+        <group>invalid_login,authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,pci_dss_10.6.1,gpg13_7.1,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,nist_800_53_AU.6,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
+      </rule>
 
-#. Paste the copied rule definition into ``/var/ossec/etc/rules/local_rules.xml``. Modify the level value, and add ``overwrite="yes"`` to indicate that this rule is overwriting an already defined rule:
+#. Paste the copied rule definition into ``/var/ossec/etc/rules/local_rules.xml``. Place it within a ``<group>`` element such as the original ``<group name="syslog,sshd,">``. Modify the level value, and add ``overwrite="yes"`` to indicate that this rule overwrites an already defined rule.
 
    .. code-block:: xml
-      :emphasize-lines: 2
-
-      <group name="syslog,sshd,">
-        <rule id="5710" level="10" overwrite="yes">
-          <if_sid>5700</if_sid>
-          <match>illegal user|invalid user</match>
-          <description>sshd: Attempt to login using a non-existent user</description>
-          <mitre>
-            <id>T1110</id>
-          </mitre>
-          <group>invalid_login,authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,pci_dss_10.6.1,gpg13_7.1,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,nist_800_53_AU.6,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
-        </rule>
-      </group>
+      :emphasize-lines: 1
```

Comment:
I think it's a good idea to include a `<group>` element this example as in the original commit.